servers/visual/rasterizer.h

```diff
@@ -930,6 +930,7 @@ class Rasterizer {
 	virtual RasterizerScene *get_scene()=0;
 
 	virtual void initialize()=0;
+	virtual void update_system_fbo(uint32_t p_system_fbo)=0;
```

Comment:
@reduz , yeah I wasn't happy with it either but while support for the fbo was added there didn't seem to be a way to change it.

One thing that I'm not sure about is if the fbo ever changes. It does seem to be written in a way that it could. I can image ios cleaning up when switching to another application and requiring re-initialisation but no idea if that actually happens.

If you can modify RasterizerStorageGLES3 the way you think it should work I can rebase this and make use of it. It also means that we split the changes needed on the renderer away from simply trying to clean up compile errors. 

Your call :)